### PR TITLE
Fix fulltext stopped

### PR DESCRIPTION
**Pull request by SuperYoko**
## What type of PR is this?
- [x] bug
- [ ] feature
- [ ] enhancement

#### Description:

build ft index is not stopable. while it's status is set to stopped. 
and then job executor return "can not stop".
It's running and no new job is allowd. 

lQLPJxbLfF_FwEXNA7jNBO2wDWmaCRG22FADTnUfAoA2AA_1261_952

## How do you solve it?

add a check in set status to early return, keep status running.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

File: src/meta/processors/job/JobDescription.cpp [hunk 1]
```diff
@@ -86,6 +86,9 @@ bool JobDescription::setStatus(Status newStatus, bool force) {
   if (JobStatus::laterThan(status_, newStatus) && !force) {
     return false;
   }
+  if (type_ == JobType::REBUILD_FULLTEXT_INDEX && newStatus == Status::STOPPED) {
```
Comment (SuperYoko) [inline src/meta/processors/job/JobDescription.cpp:89]: Wait, I'll add a stoppable judge for other type.

File: src/meta/processors/job/JobManager.cpp [hunk 2]
```diff
@@ -396,6 +397,9 @@ nebula::cpp2::ErrorCode JobManager::jobFinished(
       }
       // job has been stopped successfully, so update the job status
       return save(jobKey, jobVal);
+    } else if (code == nebula::cpp2::ErrorCode::E_JOB_NOT_STOPPABLE) {
+      // rollback the job status
+      optJobDesc.setStatus(rollbackStatus, true);
```
Comment (SuperYoko) [inline src/meta/processors/job/JobManager.cpp:402]: f***k there's risk stop block task report!

Comment (panda-sheep) [reply src/meta/processors/job/JobManager.cpp:402]: This `optJobDesc` is a temporary variable. Where to save it?

Comment (SuperYoko) [reply src/meta/processors/job/JobManager.cpp:402]: my bad.

**Comment by SuperYoko** [comment]
closed since i fix it in wrong way.

File: src/meta/processors/job/JobManager.cpp (hunk 2 above)
Comment (SuperYoko) [reply src/meta/processors/job/JobManager.cpp:402]: fixed by https://github.com/vesoft-inc/nebula/pull/4742

**Pull request closed without merging**